### PR TITLE
Add choral delete button

### PR DESCRIPTION
**Pull request by cpjk**
Push button -> choral is deleted and page refreshed with a flash message.

screenshot-2017-07-19-15-49-02

### CODE REVIEW

File: views/chorals.pug [hunk 1]
```diff
@@ -26,3 +26,6 @@ block content
           td= viewHelpers.truncateWithDots(choral.choralId, 30)
           td= choral.sampleRate
           td= choral.func
+          td
```
Comment (andreirtaylor) [inline views/chorals.pug:30]: You forgot to add a header for the delete column.

File: routes/chorals.js [hunk 2]
```diff
@@ -62,4 +62,35 @@ router.get('/new', function(req, res, next) {
   );
 });
 
+// this doesn't check to make sure the choral belong to the user but eff it for now
```
Comment (andreirtaylor) [inline routes/chorals.js]: Remove this comment

Comment (cpjk) [reply routes/chorals.js]: doesn't apply anymore

File: views/chorals.pug (hunk 1 above)
Comment (cpjk) [reply views/chorals.pug:30]: done

**Review by andreirtaylor — APPROVED** [review]

**Pull request merged**